packages/insight-viewer/src/Viewer/types.ts

```diff
@@ -1,11 +1,12 @@
+import { DataSet } from 'dicom-parser'
 import { CornerstoneImage } from '../utils/cornerstoneHelper'
 import { ProgressComponent, Viewport, OnViewportChange } from '../types'
 import { SetFrame } from '../hooks/useMultipleImages/useFrame'
 import { Interaction } from '../hooks/useInteraction/types'
 
 export type Image =
-  | (CornerstoneImage & {
-      _imageSeriesKey?: string
+  | (CornerstoneImage & { _imageSeriesKey?: string } & {
+      data: DataSet
     })
```

Comment:
```suggestion
  | (CornerstoneImage & { _imageSeriesKey?: string; data: DataSet })
```
data 에 대한 타입은 위와 같이 선언할 수 있을 것 같습니다